.env.dev

```diff
@@ -1,4 +1,5 @@
 VERSION=dev
+CODE_VERSION=0.11.2-alpha
```

Comment:
why do we need this? it seems like the trade off is without it, developing a migration is a little bit harder because you need to remember to set the versions manually (but that's kinda already true anyway). unless i'm missing something, i think this is just adding another thing that can go wrong without adding too much value.

Reply:
Since the `schema.sql` is part of the DB docker image, it will always insert the proper version string into your database (`0.11.2-alpha` for example)

But then when you bring your airbyte instance in dev mode, then your version is `dev` and you would get a version mismatch

Reply:
Ok, we can try to:
- remove the insert of the airbyte_version record from the schema.sql
- the server and sheduler read the table for that record to check versions
- if it doesn't exist:
  - the server writes it into the DB
  - the scheduler waits and retry to read again in a loop